lockable/lockable.py

```diff
@@ -84,12 +84,12 @@ def parse_requirements(requirements_str: (str or dict)) -> dict:
             return {}
         if isinstance(requirements_str, dict):
             return requirements_str
-        try:
-            return json.loads(requirements_str)
-        except json.decoder.JSONDecodeError as error:
-            # if the first char is not {, try to parse as string requirements
-            if error.colno > 1:
-                # expecting requirements_str to be a json format
+        assert isinstance(requirements_str, str), 'requirements must be string or dict'
+        requirements_str = requirements_str.strip()  # remove leading and trailing spaces
+        if requirements_str.startswith('{'):
```

Comment:
`'[]'` is valid json string also, but maybe not in this context?

Reply:
yep, not in this context for now. If in future we have such feature we can add support here as well.